apps/nextjs/src/stores/chatStore/index.ts

```diff
@@ -12,10 +15,28 @@ type ChatStore = {
   isLoading: boolean;
   stableMessages: ParsedMessage[];
   streamingMessage: ParsedMessage | null;
+  queuedUserAction: string | null;
+  isExecutingAction: boolean;
```

Comment:
I think in the context of the store, this might make more sense as `isExecutingQueuedAction`?